Formula/h/hyfetch.rb

```diff
@@ -0,0 +1,54 @@
+class Hyfetch < Formula
+  include Language::Python::Virtualenv
+
+  desc "Neofetch with LGBTQ+ pride flags + maintained neofetch"
```

Comment:
```suggestion
  desc "Fast, highly customisable system info script with LGBTQ+ pride flags"
```

or something that doesn't rely on knowing what `neofetch` is